lib/openers/evince-opener.js

```diff
@@ -19,8 +19,8 @@ const DBUS_NAMES = {
 
 function syncSource (uri, point) {
   const filePath = decodeURI(url.parse(uri).pathname)
-  atom.workspace.open(filePath).then(editor => editor.setCursorBufferPosition(point))
   atom.focus()
+  atom.workspace.open(filePath).then(editor => editor.setCursorBufferPosition(point))
```

Comment:
It probably doesn't really matter. Focusing first gives some visual cues to the movement of the cursor, though.